client/state/happychat/selectors.js

```diff
@@ -42,15 +42,15 @@ export const getHappychatTranscriptTimestamp = state => (
  * @return {String} current state value
  */
 export const getHappychatConnectionStatus = createSelector(
-	state => state.happychat.connectionStatus
+	state => get( state, 'happychat.connectionStatus' )
 );
 
 export const isHappychatConnectionUninitialized = state => getHappychatConnectionStatus( state ) === 'uninitialized';
 
 export const isHappychatClientConnected = state => getHappychatConnectionStatus( state ) === 'connected';
 
 export const isHappychatChatAssigned = createSelector(
-	state => state.happychat.chatStatus === HAPPYCHAT_CHAT_STATUS_ASSIGNED
+	state => get( state, 'happychat.chatStatus' ) === HAPPYCHAT_CHAT_STATUS_ASSIGNED
```

Comment:
Hm, my instinct is no — it's probably better to get `undefined` if the state tree isn't set. I worry that setting the default here, and also having it set in the reducer, means that we're defining a default value for the same state item in two places, and it would be easy to get out of sync.

What do you think?